packages/capnp-rpc-unix/capnp-rpc-unix.1.2.3/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+synopsis:
+  "Cap'n Proto is a capability-based RPC system with bindings for many languages"
+description:
+  "This package contains some helpers for use with traditional (non-Unikernel) operating systems."
+maintainer: "Thomas Leonard <talex5@gmail.com>"
+authors: "Thomas Leonard <talex5@gmail.com>"
+license: "Apache-2.0"
+homepage: "https://github.com/mirage/capnp-rpc"
+doc: "https://mirage.github.io/capnp-rpc/"
+bug-reports: "https://github.com/mirage/capnp-rpc/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "capnp-rpc-net" {= version}
+  "cmdliner" {>= "1.1.0"}
+  "cstruct-lwt"
+  "astring"
+  "fmt" {>= "0.8.7"}
+  "logs"
+  "extunix"
+  "base64" {>= "3.0.0"}
+  "dune" {>= "3.0"}
+  "alcotest" {>= "1.6.0" & with-test}
+  "alcotest-lwt" { >= "1.6.0" & with-test}
+  "mirage-crypto-rng-lwt" {>= "0.11.0"}
+  "mdx" {>= "2.2.1" & with-test}
+  "lwt" {>= "5.6.1"}
+  "asetmap" {with-test}
+]
+conflicts: [
+  "jbuilder" {with-test}
```

Comment:
```suggestion
  "jbuilder"
```
`with-test` is not defined in `conflicts` so it reduces into `"jbuilder" {false}`. I'm not sure why it works for you on OCaml-CI. I'm guessing this is a bug in it.

Reply:
Removing the conflict and setting the `re >= 1.9.0` lower bound should get us the same result.

Reply:
maybe but in this case I believe the conflict is the correct solution here as it's trying to avoid dune showing warnings that only happen when libraries it's using use jbuilder.